cartridges/int_algolia/cartridge/scripts/algolia/lib/algoliaContentConfig.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+var defaultAttributes = ['id', 'name', 'description', 'url', 'body', 'classificationFolder', 'page', 'order'];
+
+var attributeConfig = {
+    body: {
+        attribute: 'body',
+        localized: true
+    },
+    classificationFolder: {
+        attribute: 'classificationFolder.ID',
+        localized: false
+    },
+    description: {
+        attribute: 'description',
+        localized: true
+    },
+    folders: {
+        attribute: 'folders',
+        localized: false
+    },
+    id: {
+        attribute: 'ID',
+        localized: false
+    },
+    name: {
+        attribute: 'name',
+        localized: true
+    },
+    order: {
+        attribute: 'order',
+        localized: false
+    },
+    online: {
+        attribute: 'online',
+        localized: false
+    },
+    onlineFlag: {
+        attribute: 'onlineFlag',
+        localized: false
+    },
+    page: {
+        attribute: 'page',
+        localized: false
+    },
+    pageDescription: {
+        attribute: 'pageDescription',
+        localized: true
+    },
+    pageKeywords: {
+        attribute: 'pageKeywords',
+        localized: true
+    },
+    pageMetaTags: {
+        attribute: 'pageMetaTags',
```

Comment:
This is an Array, it won't work as is. I'd remove the complex fields like that for now.

Reply:
Done